cartography/intel/semgrep/__init__.py

```diff
@@ -20,4 +23,13 @@ def start_semgrep_ingestion(
     if not config.semgrep_app_token:
         logger.info('Semgrep import is not configured - skipping this module. See docs to configure.')
         return
-    sync(neo4j_session, config.semgrep_app_token, config.update_tag, common_job_parameters)
+
+    # fetch and load the Semgrep deployment
```

Comment:
I don't love having this block of code live here, but I'm not sure what a better alternative is. I could add a `sync_deployment` function in `intel/semgrep/deployment.py`, but it would need to return the deployment ID to be passed to the other sync functions, and it seems like the existing pattern is to have sync functions not return any data?